guides/training_with_built_in_methods.py

```diff
@@ -19,12 +19,12 @@
 ## Introduction
 
 This guide covers training, evaluation, and prediction (inference) models
-when using built-in APIs for training & validation (such as `model.fit()`,
-`model.evaluate()`, `model.predict()`).
+when using built-in APIs for training & validation (such as `Model.fit()`,
+`Model.evaluate()` and `Model.predict()`).
```

Comment:
It's a fine change, but if we're going to switch to this convention we should adopt it everywhere.

Reply:
Good idea, I'll scan the other docs. Thanks @fchollet